authors/apps/articles/models.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+from authors.apps.authentication.models import User
+from django.utils.text import slugify
```

Comment:
Kindly change local imports to be done after 3rd party library imports. This is so that we can comply with PEP8 standards on imports.

Reply:
Noted